### ISSUE TITLE
Small issue with the group management

### ISSUE THREAD

**Issue by andriish** [indico/indico#4872]
Hi all , here is a small issue with the group management

**Describe the bug**
If the group contains deleted users, one cannot add a new user to the group.
The operation seems to be successful, but the user does not appear in the list.
The error message pops out only randomly and it is very easy to overlook it.

**To Reproduce**
1.  Merge or delete users from some group
2.  Try to add a new user to that group.
3.  Don't  see any error.
4.  The user does not appear in the list of group members.

**Expected behavior**
Basically, a more visible error should be issued.

**Comment by ThiefMaster** [comment]
Merging users also merges group membership and removes the old user from all local groups, so I would not expect this to happen in case of merges - and during regular usage that's the only time when users are actually deleted...

In any case, we show an icon next to any user that's deleted so it should be rather straightforward to remove those before saving:

![image](https://user-images.githubusercontent.com/179599/114859086-8db09480-9dea-11eb-9b1d-a3e6a0245668.png)

**Comment by andriish** [comment]
> it should be rather straightforward to remove those before saving:
Yes, did that.  But before that in some cases, there was a small popup window with something like "user id =xxxx deleted". And no clear message that the list was not saved.

**Comment by andriish** [comment]
If there is an easy fix for the next version -- great, if not --- ok, one can close the issue.

**Comment by ThiefMaster** [comment]
> Yes, did that. But before that in some cases, there was a small popup window with something like "user id =xxxx deleted". And no clear message that the list was not saved.

That popup only happens when you tried to submit the form with deleted users in the group. 

We could possibly make the error message a bit clearer... after all, in case of a soft-deleted user we still have the details for them so we could show more than just the user id.

But in any case, if a field's label turns red instead of a success message, there's a good chance the data wasn't saved.

**Comment by andriish** [comment]
Outdated, closing.
